### DISCUSSION TITLE
Renumeration plan/policy

### DISCUSSION THREAD

**Discussion by stephen-rowan** [C3ETH/c3eth-operations#79] [Governance]
## Renumeration plan/policy

We need to come up with a plan and policy on how we're compensating people for their efforts. We will have two source of revenue, and specific types of budgets with two broad classes of compensation strategy. The initial discussion was introduced in our meeting #78. 

- Revenue Sources: ETH Stakepool, Donations.
- Budget Types: Operations Budget, Proposal Specific Budgets.
- Compensation Strategies: Volunteer, Hourly contribution, Wages, Bounties.

With regard to the Eastern Townhall Operations Budget, the scope of what tasks and areas should be considered for compensation. A date for commencement of accuring effort needs to be considered too.

Other than Eastern Townhall Operations Budget, funding for the proposals which are descrete projects are mostly defined in the proposals. However we have some latitude in how and where the budgets are allocated. For instance, the Connecting Asian Voters and Proposers proposal KPIs and therefore budget can contribute to running the Eastern Townhall when breakout rooms specifically include organising proposers presenting to participants. i.e. IdeaFest. Likewise Connecting East Asian Enterpreneurs includes activities to support proposers which fullfils the requirements of the Connecting Asian Voters and Proposers. The On-boarding East Asia Today! can be used to help with translations of these proposals as well.

@FelixfromSwarm has already done some thinking about compensation wrt. the Eastern Townhall budget and solicited feed back on peoples roles, expectations, and commitments.

**Comment by stephen-rowan** [comment]
## Stephen Whitenstall - Notes from October 2021 on contribution
- https://github.com/C3ETH/Catalyst-Eastern-Hemisphere-Co-Ordination/issues/88

The time I spent in October 2021 in ETH meetings and THs was approx. 16 hours. At $40 per hour that would come to $640 per month.
This would cover :

Weekly Eastern Town Hall: ( Event activities )
Support of Hosts

Weekly Additions:
Weekly organising meeting – mostly me taking minutes.

Weekly C3ETH actions: ( activities besides the Eastern Town Hall events )
I have not tracked time spent on testing or developing the C3ETH website with Robert or providing documentation on C3ETH repos. To compensate for this, I have included ETH “support” hours. 

So, this currently works out at around 4 hours per week (low estimate).

Hourly rate – the $40 hourly rate is just a figure I used on F6 proposals – it does not reflect a living wage in the UK - $70 would be a minimum self-employed rate in the UK.

Future commitments
These hours are likely to be supplemented by :
-	Assisting with ETH Cohort reporting (Auditability)
-	Infrastructure development (GitHub)
-	Documentation

It would be useful for me to commit to a certain number of hours per week for Auditability and GitHub. Maximum of 8 hours per week = 1 day.

So that would come to approx. 32 hours per month at $40 hour = $ 1240 per month.

**Comment by YanTirta** [comment]
So I've started offering Eastern Town Hall translation services to 2 (two) relations of mine. Just to see how he would react, his initial reaction is that  he is interested n would keep it in mind. I've also shared our services to a PR of a 5-star hotel just to get her pain points regarding translations. She mentioned, traditional translation services takes too long and cost too much and is only translated into 1 (one) language. She reckoned, our services would be must useful in hotels that are situated in regions that deal with a lot of foreigners/international businesses.

Then I remembered our budget situation regarding this. Rather than asking for funds to keep this translation pipeline going on, why not test out the idea of a translation business owned by the community? Where the rate we ask to translate, covers our translation pipeline development and also the cost of reimbursing the translation service from the community yet still being very credible due to it being verified by the ETH core team. By involving the community perhaps will increase further onboarding onto Cardano.

What we need to establish is our rate. Which I reckon is in the same situation regarding our compensation. Which is, it will vary from one region to another. With that said I would like to formally ask the ETH community, what would everyone feel if I were to try more aggressively pursue this translation pipeline business model? Reckon it won't last long due to the rate of Google's development, but it would be a shame not to try exploring.

TBH my main target are YouTubers. Which coincidentally Stephen brought it up n made the gears in my brain churning again.

**Comment by wolstaeb** [comment]
@andrecahyadi has been asking great questions about the Q2 2022 Comphensation plan and it's implementation. See the comments in #191 and #193, some of the initial documentation for the process in #152. I've also sketched out some notes on how to [describe tasks and size them](https://github.com/C3ETH/c3eth-operations/wiki/Describing-Issue-Tasks). And I'm also currently working on the Community Bounty payment process #105.

With one more round of doing these payments, I think it will be a good time to start documenting the process more fully. I've asked @andrecahyadi and @zicozibu to help. 

It would also be good to start using GH discuss boards to keep the team discussion located and referenced along with all the GH documents and issues.

**Comment by andrecahyadi** [comment]
@wolstaeb @YanTirta @zicozibu 
I've updated our project boards view based on each assignee name & status, please check.
I believe this should be easier for us to monitor the team's tasks.

I follow what our friends in Catalyst Circle has done with their Prioritized Problems boards, https://github.com/orgs/Catalyst-Circle/projects/8

We just need to create & update every task in the 'All Tasks' tab, then it will be automatically shown on 'each of our names' tab.

Additionally, I add a 'Wiki' status -> 'describing issue task' as an open task, just as a guidance for the team to describe the tasks they're working on.

**Reply by wolstaeb** [reply]
@andrecahyadi that is great. Yes project boards make things a lot easier to track. However the way we are *currently* using project is much lighter weight than your organisation sugguests. 

The [C3ETH Team Compensation (Q2)](https://github.com/orgs/C3ETH/projects/11/views/1) project is for doing things related to the setting up, documenting, and execucting our compensation strategy/plan for a given quarter (Q2). It is not intended as the basket for general Eastern Townhall operations. Hence only myself, you and @zicozibu need to keep track of our activities. Creating views for the entire team is not needed. 

Each project has it's own board and when you create an issues you assign a project to it. That way we can track what activities got to what project and/or milestones.  For example here are some of the current projects:

1. [F8: Eastern Catalyst School (Q2)](https://github.com/orgs/C3ETH/projects/19/views/1)
2. [Challenge Team Catalyst F8: Grow East Asia, Grow Cardano](https://github.com/orgs/C3ETH/projects/18/views/1)
3. [Eastern Townhalls](https://github.com/orgs/C3ETH/projects/17/views/1)
4. [C3ETH Team Compensation (Q2)](https://github.com/orgs/C3ETH/projects/11/views/1)

**Reply by andrecahyadi** [reply]
@wolstaeb @zicozibu Hi guys,
I see, understood. 
Yeah, I've unassigned each team member from the sample tasks that I created before and deleted the 'Wiki' status.
I left the project board view there in case we might need it later. 

So, how's the compensation categorization exactly? 
Tasks in F8 Eastern Catalyst School (Q2) category are not included in the C3ETH Team Compensation (Q2)? Is that correct? 
Do they have their own compensation plan?

The same way applied to Eastern Townhalls & Challenge Team Catalyst F8: Grow East Asia, Grow Cardano categories, which are not included in the C3ETH Team Compensation (Q2).

**Comment by andrecahyadi** [comment]
@wolstaeb @zicozibu @YanTirta 
Hi guys, continue my questions above, I want to ask and need confirmation on how to categorize the tasks. 

From those 4 current projects that Robert mentioned above:
1. F8: Eastern Catalyst School (Q2)
2. Challenge Team Catalyst F8: Grow East Asia, Grow Cardano
3. Eastern Townhalls
4. C3ETH Team Compensation (Q2)

Suppose someone did these tasks during a week (examples):
- Manage the local community for a week
- GitHub administration (local community)
- GitHub administration (ETH)
- GitHub administration Eastern Catalyst School
- Update website content
- Review website content
- Meeting issue management - task planning workshop (ETH)
- Catalyst Reporting - April progress
- Event Townhall - preparation breakout room
- Content Translation - Fund Challenges

How do we put each task in those 4 project categories? 
And then, after we put the size on each task, will there be anyone come to check whether those tasks have the correct sizes?
Does the person who checks the sizes also get compensated? Supposed those tasks above are 100% DRI from the doer himself (not from another person or the checker). 

Looking forward to your explanation. Thank You, guys :)
